src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const fieldBlock = document.querySelector('.field tbody');
+let field = Array.from(fieldBlock.querySelectorAll('tr'));
+
+document.addEventListener('DOMContentLoaded', () => {
+  fieldBlock.innerHTML = '';
```

Comment:
Clearing the `fieldBlock` innerHTML on DOMContentLoaded might remove the initial table structure. Ensure that the initial table setup is handled correctly if this is intended.